frontend/src/components/sections/edit-section.jsx

```diff
@@ -52,75 +53,67 @@ const EditSection = () => {
 
   return (
     <>
-      <Header text="Edit scene" />
+      <Header text="Edit scene AABB" />
```

Comment:
Why don't we use the `SceneLayout` here?

Reply:
`SectionLayout` renamed to `SectionLayoutWithSidebar`